samples/sp_whoIsActive/package.json

```diff
@@ -0,0 +1,228 @@
+                "container": {
+                    "grid-container": [
+                        {
+                            "name": "Tasks",
+                            "row": 0,
+                            "col": 0,
+                            "colspan":2,
+                            "widget": {
+                                "tasks-widget": [
+                                    "sp_whoisactive.create", "sp_whoisactive.findBlockLeaders"
+                                ]
+                            }
+                        },
+                        {
+                            "name": "CPU Usage",
+                            "row":1,
+                            "col":0,
+                            "colspan":2,
+                            "widget": {
+                                "sp_whoisactive-CPU-usage": {
+                                }
+                            }
+                        },
+                        {
+                            "name": "CPU Delta",
+                            "row":1,
+                            "col":1,
+                            "colspan":2,
+                            "widget": {
+                                "sp_whoisactive-CPU-delta": {
+                                }
+                            }
+                        },
+                        {
+                            "name": "Memory Usage",
+                            "row":2,
+                            "col":0,
+                            "colspan":2,
+                            "widget": {
+                                "sp_whoisactive-memory-usage": {
+                                }
+                            }
+                        },
+                        {
+                            "name": "Memory Delta",
+                            "row":2,
+                            "col":1,
+                            "colspan":2,
+                            "widget": {
+                                "sp_whoisactive-memory-delta": {
+                                }
+                            }
+                        }
+                    ]
+                }
+            }
+        ],
+        "snippets": []
+    },
+    "scripts": {
+        "build": "gulp build",
+        "compile": "gulp compile",
+        "watch": "gulp watch",
+        "postinstall": "node ./node_modules/vscode/bin/install"
+    },
+    "dependencies": {
+        "fs-extra": "^5.0.0",
+        "handlebars": "^4.0.11",
+        "vscode-nls": "2.0.2"
+    },
+    "devDependencies": {
+        "@types/handlebars": "^4.0.11",
+        "@types/mocha": "^2.2.42",
+        "@types/node": "^7.0.43",
+        "child-process-promise": "^2.2.1",
+        "del": "^3.0.0",
+        "gulp": "^4.0.0",
+        "gulp-color": "0.0.1",
+        "gulp-sourcemaps": "^2.6.4",
+        "gulp-tslint": "^6.0.2",
+        "gulp-typescript": "^3.2.4",
+        "pm-mocha-jenkins-reporter": "^0.2.6",
```

Comment:
Are all these necessary? Given lack of tests you can definitely remove the jenkins-reporter. I'd recommend cleaning these up if possible.